index.html

```diff
@@ -44,6 +44,16 @@ <h2 class="instructionsOverlayTitle">How to play the game?</h2>
                 </div>
                 <button id="startGameButton">START!</button>
             </div>
+                <div id="displayEndGameContainer">
+                    <p>Far out brah! That was totally radical &#129305;</p>
+                    <p>Enter your name to save your time! Catch you on flip!</p>
+                    <form id="formContainer">
+                        <label for="nameField">Name:</label>
+                        <input id="nameField" type="text" name="Name" maxlength=3>
+                        <input class="saveButton" type="button" onclick="getName(this.form)" value="Save!">
+                        <input class="playButton" type="button" onclick="history.go(0)" VALUE="Play again!">
```

Comment:
unfortunately inline js event listener is bad, it needs to be added in the js file